cloud/filestore/libs/vfs_fuse/loop.cpp

```diff
@@ -601,6 +609,25 @@ class TFileSystemLoop final
 
                     s.SetValue();
                 });
+
+            // We need to cleanup HandleOpsQueue file and directories
+            if (p->HandleOpsQueueInitialized) {
```

Comment:
since session is destroyed asynchronously, you need to do the cleanup in the handler that is invoked after the session is destroyed